src/index.ts

```diff
@@ -211,8 +212,13 @@ export interface RetryOptions {
   retryableErrorFn?: (err: ApiError) => boolean;
 }
 
+export interface GoogleInnerError {
```

Comment:
Any reason for the additional interface as opposed to just declaring `errors` (below) as `string[]`?

Reply:
yeah the compiler won't let me. It fails to map the logic below to `ApiError` without this